CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master
 
 * Fixed an issue where the CarPlay navigation map’s vanishing point and user puck initially remained centered on screen, instead of accounting for the maneuver panel, until the navigation bar was shown. ([#1856](https://github.com/mapbox/mapbox-navigation-ios/pull/1856))
+* Updated views on `BaseInstructionsBannerView` to be public to help with being able to build custom instruction banners. This is considered experiential and could change at any time.
```

Comment:
jazzy is configured to hide any undocumented method, so I think we can generally fall back to the same policy that the iOS map SDK (and Apple’s own APIs) follow: undocumented methods carry no inherent backward compatibility guarantees. In fact, the containing class is still marked `:nodoc:`, preventing any documentation from appearing even if we added documentation comments to these properties.

To avoid leading developers to use unsupported APIs, I removed this changelog entry in e7c17f019205e2eb9cee86574187bdec71250774.